plugins/libplugin.c

```diff
@@ -547,7 +549,7 @@ static struct command_result *handle_init(struct command *init_cmd,
 					  take(param_obj),
 					  &rpc_conn,
 					  ".allow-deprecated-apis"),
-				"true");
+					  "true");
```

Comment:
Also, the original, indent of this line is correct; it is not a continuation of the argument list of the previous line.